foundry.toml

```diff
@@ -39,7 +39,7 @@ nova = "${ETH_NODE_URI_NOVA}"
 # testnets
 goerli = "${ETH_NODE_URI_GOERLI}"
 mumbai = "${ETH_NODE_URI_MUMBAI}"
-consensys-zkevm-testnet = "${ETH_NODE_URI_CONSENSYS_ZKEVM_TESTNET}"
```

Comment:
Would be good to also adjust the env.example with the new network

Reply:
Added